lib/basic/8_increment_decrement/increment_decrement.dart

```diff
@@ -0,0 +1,20 @@
+class Increment {
+  jalankan() {
+    var a = 0;

```

Comment:
Mungkin lebih baik menggunakan strong type data daripada menggunakan var. untuk semua var mungkin lebih baik diganti int saja ya, karena datanya udah jelas menggunakan int.
```suggestion
    int a = 0;

```

Reply:
Untuk membiasakan diri juga dengan tipe data 👍